MH-FiorellaTheme5/source/categories.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="container">
```

Comment:
Why you have this file in this Pull Request?

Reply:
Oh... I did a checkout in master (MH-theme5), and when a did a pull that changes were brought.